shopping-webclient/src/components/checkout/stripe/Stripe.vue

```diff
@@ -52,15 +62,94 @@ export default {
   },
 
   methods: {
-    async pay() {
+    async pay(event) {
       // createToken returns a Promise which resolves in a result object with
       // either a token or an error key.
       // See https://stripe.com/docs/api#tokens for the token object.
       // See https://stripe.com/docs/api#errors for the error object.
       // More general https://stripe.com/docs/stripe.js#stripe-create-token.
       try {
-        const data = await Stripe.createToken();
-        this.$emit('pay', data.token);
+        // const data = await Stripe.createToken();
+        // this.$emit('pay', data.token);
+        event.target.disabled = true;
```

Comment:
Instead of directly accessing `event.target.disabled`, can you utilize the `complete` variable or create one more variable to handle this logic and pass it as a `disabled` prop in the `<b-btn>` component?

Reply:
This was actually a quick fix to disabling a button. Perhaps, you could merge this PR into a new branch where the both of us could colab on such changes.